plugins/intel-me/meson.build

```diff
@@ -0,0 +1,15 @@
+if get_option('plugin_amt').disable_auto_if(host_machine.system() != 'linux').allowed()
+cargs = ['-DG_LOG_DOMAIN="FuPluginIntelMkhi"']
```

Comment:
this seems inconsistent with rest of the naming

Reply:
Fixed.